src/main/java/figures/Round.java

```diff
@@ -0,0 +1,32 @@
+package figures;
+
+public class Round extends Shape {
+    private final String NAME = "Round";
```

Comment:
don't see value to have this as separate variable 
but if you really need it, constant has different declaration 
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names